### PR TITLE
step 2 to fix 132

### PR DESCRIPTION
**Pull request by ocefpaf**
xref.: https://github.com/ioos/ioos_code_lab/issues/132#issuecomment-1485683515

- [x] add note on `jupyter-lab` and `jupyter nbclassic` in this file.

### CODE REVIEW

File: jupyterbook/content/ioos_installation_conda.md [hunk 1]
```diff
@@ -168,6 +172,10 @@ Check out [Travis Oliphant's blog piece](http://technicaldiscovery.blogspot.com/
 
 ### What to do when everything seems to be broken?
 
+For example, ff you are seeing kernel erros like the one below.
```
Comment (MathewBiddle) [inline jupyterbook/content/ioos_installation_conda.md:175]: **if** you are seeing kernel **errors** like the one below.

File: jupyterbook/content/ioos_installation_conda.md [hunk 2]
```diff
@@ -118,6 +118,10 @@ Once the environment is done building, you can activate it by typing:
 conda activate IOOS
 ```
 
+Now you can start hacking the notebook with `jupyter nbclassic`,
+for the old style notebooks,
+ or `jupyter-lab` for the new IDE interface.
```
Comment (MathewBiddle) [inline jupyterbook/content/ioos_installation_conda.md:123]: This is not available in our env:
```
(base) C:\>conda activate ioos
(ioos) C:\>jupyter-lab
'jupyter-lab' is not recognized as an internal or external command,
operable program or batch file.
```

Comment (ocefpaf) [reply jupyterbook/content/ioos_installation_conda.md:123]: It is added in this PR.

Comment (MathewBiddle) [reply jupyterbook/content/ioos_installation_conda.md:123]: duh - sorry for that oversight

Comment (ocefpaf) [reply jupyterbook/content/ioos_installation_conda.md:123]: To be fair I also thought it was already there.

File: jupyterbook/content/ioos_installation_conda.md [hunk 3]
```diff
@@ -172,7 +172,7 @@ Check out [Travis Oliphant's blog piece](http://technicaldiscovery.blogspot.com/
 
 ### What to do when everything seems to be broken?
 
-For example, ff you are seeing kernel erros like the one below.
+For example, if you are seeing kernel erros like the one below.
```
Comment (MathewBiddle) [inline jupyterbook/content/ioos_installation_conda.md:175]: erros -> errors

Comment (ocefpaf) [reply jupyterbook/content/ioos_installation_conda.md:175]: Jeez, I cannot even copy-n-paste this afternoon! Sorry.

**Comment by MathewBiddle** [comment]
LGTM! Thanks for addressing these.

**Pull request merged**